main.py

```diff
@@ -91,26 +92,37 @@ def delete_label_file(self):
         label_filename = os.path.splitext(self.image_filename)[0] + ".txt"
         if os.path.exists(label_filename):
             os.remove(label_filename)
-    def load_labels(self, image_filename, existing_labels):
+
+
+    def load_labels(self, image_filename, existing_labels, rollover_label):
         #check if label file exists
         label_filename = os.path.splitext(image_filename)[0] + ".txt"
-        self.clear_labels() #delete this line if you want to have labels roll over from previous frame
-        if os.path.exists(label_filename):
-            print("Loading existing labels for", image_filename)
-            #MOVE HERE self.clear_labels() if you want to keep the previous frames labels
-            #draw labels from file onto canvas
-            with open(label_filename, "r") as yolo_label_file:
-                yolo_labels = yolo_label_file.read().splitlines()
-                for yolo_label in yolo_labels:
-                    class_index, x, y, width, height = yolo_label.split(" ")
-                    class_index = int(class_index)
-                    x, y, width, height = float(x), float(y), float(width), float(height)
-                    class_name = CLASSES[class_index]
-                    bb = [x,y, x+width, y+height]
-                    bb_id = self.create_rectangle(bb[0] * self.winfo_width(),bb[1] * self.winfo_height() ,bb[2] * self.winfo_width(),bb[3] * self.winfo_height(), fill="", outline=COLORS[class_index])
-                    new_label_text_id = self.create_text(x * self.winfo_width(), (y * self.winfo_height()) - 5, fill=COLORS[class_index], text=class_name)
-                    self.labels.append([bb,class_name,bb_id, new_label_text_id])
-        print("labels:")
+        #self.clear_labels() #delete this line if you want to have labels roll over from previous frame
+        print(f"Rollover label: {self.rollover_label}")
+      
+       
+        if self.rollover_label == False:
+            self.clear_labels()
+
+        if len(self.labels) > 0:
+            print ("labels exist, no need to overwrite")
```

Comment:
Is this meant to prevent overwrites of frames that already have labels?  For me this debug text is always firing and it is always still rolling over labels

Reply:
Consider the scenarios:
1) No images are loaded
2) An image has loaded, but no labels were created---we don't want to create the .txt file.

Reply:
@SteRMo95 It prevents overwriting when you switch back and forth between frames with rollover mode on.